projects/observability/src/shared/dashboard/data/graphql/specifiers/attribute-sort-specification.model.ts

```diff
@@ -0,0 +1,33 @@
+import { Dictionary } from '@hypertrace/common';
+import { Model, ModelProperty, STRING_PROPERTY } from '@hypertrace/hyperdash';
+import { GraphQlSortDirection } from '../../../../graphql/model/schema/sort/graphql-sort-direction';
+import { Specification } from '../../../../graphql/model/schema/specifier/specification';
+import { SpecificationBuilder } from '../../../../graphql/request/builders/specification/specification-builder';
+import { SpecificationModel } from './specification.model';
+
+@Model({
+  type: 'attribute-sort-specification',
+  displayName: 'Attribute Sort'
+})
+export class AttributeSortSpecificationModel extends SpecificationModel<Specification> {
```

Comment:
Confused about why we need this to extend the specification model?

Reply:
You are the man. Thanks for calling this out. Its unnecessary for this model.